lib/tools/apk-utils.js

```diff
@@ -53,19 +53,21 @@ apkUtilsMethods.isAppInstalled = async function isAppInstalled (pkg) {
  *
  * @param {string} uri - The name of URI to start.
  * @param {string} pkg - The name of the package to start the URI with.
+ * @param {boolean} waitForLaunch - Optional parameter of waitForLaunch caps to add wait in adb command
  */
-apkUtilsMethods.startUri = async function startUri (uri, pkg) {
+apkUtilsMethods.startUri = async function startUri (uri, pkg, opts = { waitForLaunch: true }) {
   if (!uri || !pkg) {
     throw new Error('URI and package arguments are required');
   }
 
-  const args = [
-    'am', 'start',
-    '-W',
-    '-a', 'android.intent.action.VIEW',
-    '-d', uri.replace(/&/g, '\\&'),
-    pkg,
-  ];
+  const args = ['am', 'start'];
+  if (opts.waitForLaunch) {
+    args.push('-W');
+  }
+  args.push('-a', 'android.intent.action.VIEW',
+       '-d', uri.replace(/&/g, '\\&'),
```

Comment:
Could you change 2 indentation style?

Reply:
@KazuCocoa you mean both args should be in same vertical line. 
Eslint didn't suggest me this so I missed.

Reply:
It is throwing me error when I am trying to change

Reply:
ah, okay